ext/intl/timezone/timezone_methods.cpp

```diff
@@ -83,9 +83,7 @@ U_CFUNC PHP_FUNCTION(intltz_from_date_time_zone)
 
 	tzobj = Z_PHPTIMEZONE_P(zv_timezone);
 	if (!tzobj->initialized) {
-		intl_error_set(NULL, U_ILLEGAL_ARGUMENT_ERROR,
-			"DateTimeZone object is unconstructed",
-			0);
+		zend_throw_error(NULL, "DateTimeZone object is unconstructed");
```

Comment:
I think the way to test this is by creating the object via Reflection and using the method the constructs the object without calling the constructor